src-v2/Not/Not.Blazor/TM/Models/NotListModel.cs

```diff
@@ -22,6 +22,11 @@ public static IEnumerable<NotListModel<T>> FromEntity<T>(IEnumerable<T> values)
             yield return new NotListModel<T>(value);
         }
     }
+
+    public static NotListModel<T> GetEmptyValue<T>()
```

Comment:
Is it getting an empty value or creating an empty list?

Reply:
It does create an empty object with `Value=null;`